linux/network/systemd.sls

```diff
@@ -0,0 +1,33 @@
+{%- from "linux/map.jinja" import network with context %}
+{%- if network.enabled and grains.get('init', None) == 'systemd' %}
+
+{%- if network.systemd is mapping %}
+{%- for config_type, configs network.systemd.iteritems() %}
+{%- for config_name, config in configs.iteritems() %}
+linux_network_systemd_networkd_{{ config_type }}_config_{{ config_name }}:
+  file.managed:
+    - name: /etc/systemd/network/{{ config_name }}-salt-network.{{ config_type }}
+    - source: salt://linux/files/systemd.conf
```

Comment:
you should make better name of that template (systemd-network.conf or something)